README.md

```diff
@@ -43,6 +43,135 @@ changed, which should be synced into the container. Because
 will happen. After this the container will be restarted and
 the `CMD` step will run again.
 
+## Webpack-dev-server/nodemon/my custom monitor is faster!
+
+We agree! Special case programs can be extremely fast, and
+livepush does not stop you from using them from within a
+container. To enable this, you can use live directives:
+
+### dev-cmd-live
+
+A live cmd is the command that gets executed in the
+container when this container is running with livepush. It
+overrides the existing `CMD` in the Dockerfile. This is
+achieved with a pre-process step on the Dockerfile.
```

Comment:
```suggestion
achieved with a preprocess step on the Dockerfile.
```

Original is fine, but you have used this variant in a couple of other places in the doc